lib/thread.c

```diff
@@ -44,13 +44,37 @@ DEFINE_MTYPE_STATIC(LIB, THREAD_STATS,  "Thread stats")
 /* Relative time, since startup */
 static struct hash *cpu_record = NULL;
 
-static unsigned long
+unsigned long
 timeval_elapsed (struct timeval a, struct timeval b)
 {
   return (((a.tv_sec - b.tv_sec) * TIMER_SECOND_MICRO)
 	  + (a.tv_usec - b.tv_usec));
 }
 
+void
+timeval_copy (struct timeval *a, struct timeval b)
```

Comment:
I don't see this being used much, especially when it could be accomplished in one line with a memcpy